app/src/main/java/com/example/ithaca_transit_android_v2/states/RouteCardState.kt

```diff
@@ -1,22 +1,23 @@
 package com.example.ithaca_transit_android_v2.states
 
 import com.example.ithaca_transit_android_v2.models.Route
+import com.example.ithaca_transit_android_v2.models.RouteOptions
 
 // Parent Class
-sealed class RouteCardState (val routes: List<Route> = listOf())
+sealed class RouteCardState(val routes: RouteOptions? = null)
```

Comment:
if `routes` is not required for some route card states, why don't we just remove this param from the parent?